_posts/developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -112,398 +108,393 @@ Node.js installation.
+   1. First, click the `Read Profile` button. 
+      
+      You should see the name you entered when you registered your Pod. 
 
-   * Now let's update your name. Simply enter a new name in the '2. Write your
-   name' textfield, and click the 'Write to Profile' button.
-       * You should see the message:
-           ```
-           Wrote [<your new name>] as name successfully!
-           ```
+   1. To update your name, enter a new name in the `2. Write your name`
+        textfield, and click the `Write to Profile` button. You should see the message:
+        
+        ```
+        Wrote [<your new name>] as name successfully!
+        ```
          
-   * Now let's verify that your profile really was updated by clicking again on
-   the 'Read Profile' button.
-       * You should see the updated name displayed!
+   1. Verify that your profile was updated by clicking on
+        the `Read Profile` button again. You should see the updated name displayed!
 
    You can also read the public profiles of anyone else in the world with a
-   Solid Pod by simply entering their WebID into the text box of section 3;
-    e.g., `https://docs-example.inrupt.net/profile/card#me` (**NOTE:** If you do try that example WebID, you should see the name
-   `Changing Docs Example`!).
+   Solid Pod.  Enter the WebID of the person whose profile you wish to read;
+    e.g., `https://docs-example.inrupt.net/profile/card#me`.
 
-10. Exit the Application
-
-    * To exit the application, simply stop the `npx parcel` process; e.g.,
+1. Exit the Application. To exit the application, stop the `npx parcel` process; e.g.,
      `Ctrl-C`.
 
 
 ## Additional Information
 
 #### API Documentation
 
+For more information on the functions:
+
 - [solid-client API](https://docs.inrupt.com/developer-tools/api/javascript/solid-client/README.html)
+  - [getSolidDataset()](https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/_resource_soliddataset_.html#getsoliddataset)
+  - [getThing()](https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/_thing_thing_.html#getthing)
+  - [saveSolidDatasetAt()](https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/_resource_soliddataset_.html#savesoliddatasetat)
+  - [setStringNoLocale()](https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/_thing_set_.html#setstringnolocale)
+  - [setThing()](https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/_thing_thing_.html#setthing)
 
-- [solid-client-authn-browser API](https://docs.inrupt.com/developer-tools/api/javascript/solid-client-authn-browser/README.html)
 
+- [solid-client-authn-browser API](https://docs.inrupt.com/developer-tools/api/javascript/solid-client-authn-browser/README.html)
 
-#### React SDK Availability
+  - [Session.login()](https://docs.inrupt.com/developer-tools/api/javascript/solid-client-authn-browser/classes/_session_.session.html#login)
```

Comment:
Wasn't sure whether we should link to the Inrupt docs. Since it had the React docs link, I did.  But can also get rid of both.